### ISSUE TITLE
Schema update: design schema for drug combination data

### ISSUE THREAD

**Issue by sgosline** [PNNL-CompBio/coderdata#53]
Identify 3-4 use cases of algorithms that predict drug combinations, determine best format for creating an experiments.csv file that has drug combination data. I'm guessing AUCs will not be useful.

**Comment by sgosline** [comment]
This will look very similar to the sanger model as their code pre-calculates synergy. 
```
          range: CellPerturbation
  Combinations:
      description: This table holds the drug combination data. It is primarily pulled from https://gdsc-combinations.depmap.sanger.ac.uk/documentation where I can.
      slots:
        - improve_sample_id
        - time
        - time_unit
      attributes:
        first_drug:
          slot_uri: improve_drug_id
          description: improve_drug_id of the first drug
        second_drug:
          slot_uri: improve_drug_id
          description: improve_drug_id of the second drug
        first_concentration:
          description: Concentration of first drug 
          range: float
        second_concentration:
          description: Concentration of second drug
          range: float
        concentration_unit:
          description: Unit of measurement of both drugs
	synergy measurement: 
	  description: Description of synergy measurement
        synergy value:
          description: Measurement of synergy between drugs
          range: float
	
```

**Comment by jjacobson95** [comment]
Schema updates to consider:

Samples, model_type may include:
- Saliva
- Blood ?
- Not Reported

Mutations, Variant types should include:
- 3'Flank (hcmi)
- Splice_Region (in addition to Splice_Site, HCMI)
- Translation_Start_Site (typo fix from "Translationn_Start_Site")
- Frameshift_Variant (BeatAML)
- Protein_Altering_Variant (BeatAML)
- Should splice_acceptor_variant and splice_donor_variant both map to the current Splice_Site? (Both are listed in BeatAML)

Experiments:
- Are we interested in r2fit?

We can discuss soon.

**Comment by sgosline** [comment]
This is closed by #105
